src/extension.ts

```diff
@@ -158,7 +158,7 @@ async function runRemark(document: vscode.TextDocument, range: vscode.Range): Pr
 		if (result.messages.length !== 0) {
 			let message = '';
 
-			result.messages.forEach((value: string) => {
+			result.messages.forEach((value) => {
```

Comment:
in this case, `value` is a [`VFileMessage`](https://github.com/vfile/vfile-message/blob/04d001493de202ffad70110bb194739a91f312d3/types/index.d.ts#L9-L93)